backend/grafana-dashboards/backend.json

```diff
@@ -0,0 +1,494 @@
+{
+  "description": "ARO HCP Backend Component",
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 0,
+  "id": 178,
+  "links": [],
+  "panels": [
+    {
+      "datasource": {
+        "default": false,
+        "type": "prometheus",
+        "uid": "${datasource}"
+      },
+        "description": "Backend Health where # number of replicas should equal the value",
+        "fieldConfig": {
+          "defaults": {
+            "custom": {
+              "fillOpacity": 10,
+              "gradientMode": "hue",
+              "showPoints": "never"
+            }
+          }
+        },
+        "gridPos": {
+          "h": 10,
+          "w": 24,
+          "x": 0,
+          "y": 0
+        },
+        "id": 1,
+        "options": {
+          "legend": {
+            "calcs": [],
+            "displayMode": "list",
+            "placement": "bottom",
+            "showLegend": true
+          },
+          "tooltip": {
+            "mode": "single",
+            "sort": "none"
+          }
+        },
+        "targets": [
+          {
+            "datasource": {
+              "type": "prometheus",
+              "uid": "${datasource}"
+            },
+            "editorMode": "code",
+            "expr": "count(max without(prometheus_replica) (backend_health{cluster=\"$cluster\"}))",
```

Comment:
Shouldn't compare the sum of backend_health with the count of backend_health?

Reply:
Updated along with the frontend dashboard